draft/spec/index.html

```diff
@@ -830,10 +830,9 @@ <h2>Fixity</h2>
     <section id="inventory-digest">
         <h2>Inventory Digest</h2>
         <p>
-            Every occurrence of an inventory file <span id="E058">MUST</span> have an accompanying sidecar file stating
-            its digest. This sidecar file <span id="E092">MUST</span> be of the form
```

Comment:
Does the corresponding validation code also go away? 
   - E092 - https://github.com/OCFL/spec/blob/master/validation/validation-codes.md

Reply:
Yes _in theory_, but you'll note that E092 isn't in the set of validation codes yet, they haven't been completed that far

Reply:
Hmm... I may be looking at the wrong file, but searching for "E092" in https://github.com/OCFL/spec/blob/master/validation/validation-codes.md shows an entry between "E058" and "E059".

Reply:
It does, how odd! I looked for the last entry which was just E068... have created https://github.com/OCFL/spec/pull/454